src/operator/concatAll.ts

```diff
@@ -12,6 +11,6 @@ import {MergeAllOperator} from './mergeAll-support';
  *
  * @returns {Observable} an observable of values merged from the incoming observables.
  */
-export function concatAll<T>(): Observable<T> {
+export function concatAll<T>(): T {
```

Comment:
what we're really looking for is something like

```typescript
concatAll<T extends Observable<InnerT>, InnerT>(): Observable<InnerT>
```

which sadly isn't supported with typescript, and we might have to get away with just docs here.  If we make it `Observable<T>` then true return type would show up as `Observable<Observable<T>>`